main.go

```diff
@@ -2,6 +2,11 @@ package main
 
 import "fmt"
 
-func main(){
-	fmt.Println("This is a test project named mantis")
-}
\ No newline at end of file
+// SayHello to mantis
+func say_hello() {
```

Comment:
Use camel-case format for method name

Reply:
```suggestion
func sayHello() {
```

Reply:
```suggestion
func sayHello() {
```

Reply:
You should use camel case for method name,
as follows:

```suggestion
func sayHello() {
```

Reply:
```suggestion
/*
 * Use camel case as function name
 * No published method should start with character in lower case.
 */
func sayHello() {
```